src/controllers/dynakube/activegate/internal/statefulset/statefulset.go

```diff
@@ -135,7 +135,7 @@ func (statefulSetBuilder Builder) buildBaseContainer() []corev1.Container {
 			ProbeHandler: corev1.ProbeHandler{
 				HTTPGet: &corev1.HTTPGetAction{
 					Path:   "/rest/health",
-					Port:   intstr.IntOrString{IntVal: 9999},
```

Comment:
Why not using the const here? `HttpsContainerPort`

Reply:
that is the change here 😬 
(🔴 == old ; 🟢 == new)

Reply:
brain lag :D